### DISCUSSION TITLE
Streaming from Retroarch help needed

### DISCUSSION THREAD

**Discussion by RobertMCortese** [bluenviron/mediamtx#3580] [Questions]
### Question

Hello,

I'd like to stream from retroarch to my mediaMTX install. I can get retroarch to connect to the mediaMTX server, but I get this error.
2024/07/26 08:52:25 INF [RTMP] [conn [::1]:9182] opened
2024/07/26 08:52:27 INF [RTMP] [conn [::1]:9182] closed: invalid message type: 255

This is the config I'm using on the retroarch side.
vcodec = libx264
acodec = libmp3lame
pix_fmt = yuv420p
scale_factor = 2
threads = 3
video_crf = 25
video_preset = superfast
video_tune = animation
audio_global_quality = 75
sample_rate = 44100
format = flv

**Comment by aler9** [comment]
Hello, i tried streaming from Retroarch to the server and it worked flawlessly.
I used Ubuntu 22.04, FFmpeg 7.0.1, retroarch 1.19.1.

stream.cfg content:

```
vcodec = libx264
acodec = libmp3lame
pix_fmt = yuv420p
scale_factor = 2
threads = 3
video_crf = 25
video_preset = superfast
video_tune = animation
audio_global_quality = 75
sample_rate = 44100
format = flv
```

retroarch was launched with 

```
retroarch --record rtmp://127.0.0.1:1935/stream --recordconfig stream.cfg
```
